release notes/v0.35.0.md

```diff
@@ -0,0 +1,134 @@
+k6 v0.35.0 is here! :tada: It introduces several new features that nicely enhance its usability and also contains a whole lot of fixes and ongoing efforts with refactoring.
+
+In total, we have closed 14 issues. We have also branched out three new xk6 extensions during this release :star:
```

Comment:
```suggestion
In total, we have closed 14 issues. We have also branched out three new [xk6 extensions](https://k6.io/docs/misc/k6-extensions/) during this release :star:
```

Doesn't hurt to always link to the docs, some users might have never heard about xk6 :man_shrugging:

Reply:
It makes sense but now that I've added it, this actually looks strange to me as I'd think the link leads to those three extensions :sweat_smile: